### PR TITLE
Image Minification

### LINKED ISSUE #10: Image minification
Ideally on `gulp release` we should have an automated image magnification process. A google search brings up a few tools for the job.

Personally I don't have too much of a preference, but we should certainly look for something that doesn't require us to have imagemagick / graphicsmagick etc installed on our machines.

### PR DESCRIPTION
**Pull request by oliverbenns**
- Added imagemin task when looping through image asset directory.
- Fixes #10 
- Used in previous projects and seemed to work well (See [AEG](https://github.com/inlight-media/aeg/blob/master/www/gulpfile.js) )

### CODE REVIEW

File: www/tasks/assets.js [hunk 1]
```diff
@@ -27,14 +30,19 @@ gulp.task('assets', function() {
 	dirs.forEach(function(dir) {
 		var cwd = path.join(srcAssetsPath, dir);
 		var dest = path.join(destAssetsPath, dir);
-		var useSymlink = true; // @TODO: When it's release copy otherwise symlink.
```
Comment (sinakarimi) [inline www/tasks/assets.js:30]: I don't think 'useSymlink' should be removed as we rely on it later on to symlink the dest directory

Comment (oliverbenns) [inline www/tasks/assets.js:30]: Oh woops! Totally didn't mean to remove that, I had to toggle the Boolean temporarily but must have removed by mistake 🙈

**Comment by sinakarimi** [comment]
This looks good to me @oliverbenns :+1: 

Would be good to get other peoples thoughts as well.

**Comment by beaucharman** [comment]
I dig. nice stuff Ollie +1

**Pull request merged**